Dockerfile.assisted-installer-controller

```diff
@@ -11,12 +10,14 @@ COPY go.sum go.sum
 RUN go mod download
 
 COPY . .
-RUN make controller
+RUN TARGETPLATFORM=$TARGETPLATFORM make controller
 
 FROM quay.io/centos/centos:stream8
+ARG TARGETPLATFORM
 
 RUN yum -y install make gcc unzip wget curl rsync && yum clean all
+RUN case $TARGETPLATFORM in linux/amd64 | "") arch=x86_64 ;; linux/arm64) arch=aarch64 ;; *) echo Invalid platform $TARGETPLATFORM && exit 1 ;; esac ; \
+   curl -k https://mirror.openshift.com/pub/openshift-v4/$arch/clients/ocp/4.11.3/openshift-client-linux.tar.gz | tar xfz - -C /usr/bin oc && chmod +x /usr/bin/oc
```

Comment:
don't we have multi arch images for oc cli? Otherwise, we will end up with the same issues described in https://github.com/openshift/assisted-service/pull/4286

Reply:
Good point.
Unfortunately, that repo **origin-cli** is not multi arch.  Didn't find another that is multi-arch.

Reply:
Can you ask in multi-arch channel? I think we need to open ticket for them

Reply:
+1 to @tsorya. Lets not reinvent the wheel here - lets ask on slack

Reply:
@romfreiman but i think we can push it as is and open relevant tickets wdyt?

Reply:
i think that asking takes exactly 10 seconds

Reply:
BTW, the problem in the PR https://github.com/openshift/assisted-service/pull/4286 was that we used dev-preview

Reply:
Changed to use [quay.io/openshift/origin-cli-artifacts](http://quay.io/openshift/origin-cli-artifacts)  according to suggestion from https://coreos.slack.com/archives/CFFJUNP6C/p1662470502219889?thread_ts=1662449204.007179&cid=CFFJUNP6C